formulas/functions/statistical.js

```diff
@@ -215,7 +215,23 @@ const StatisticalFunctions = {
 
   MAX: () => {},
 
-  MAXA: () => {},
+  MAXA: (...values) => {
+    const arr = [];
+    arr.push(...values.map(a => a.value))
+    const flatArr = FormulaHelpers.flattenDeep(arr)
+    const updatedArr = flatArr.map(a => {
+      if(a === true)
+        return 1
+      if(!isNaN(parseFloat(a)))
+        return parseFloat(a)
+      if(!isNaN(parseInt(a)))
+        return parseInt(a)
+      return 0
```

Comment:
If the data we are checking contains _all_ negative numbers, this line will cause the function to return `0`, which would not be the max.

```suggestion
      return -Infinity
```

Reply:
I changed it a little, Instead of returning a number, I return null if its an invalid input then filter out all the nulls

Reply:
const updatedArr = flatArr.map(a => {
      if(a === true)
        return 1
      if(!isNaN(parseFloat(a)))
        return parseFloat(a)
      if(!isNaN(parseInt(a)))
        return parseInt(a)
      return null
    })
    const finalArr = updatedArr.filter((a) => a != null)